src/table/src/TableRow.js

```diff
@@ -67,22 +72,32 @@ const TableRow = memo(
       onKeyPress(e)
     }
 
-    const themedClassName = theme.getRowClassName(appearance, intent)
+    const { className: themedClassName, ...boxProps } = useStyleConfig(
+      'TableRow',
+      { appearance, intent },
+      pseudoSelectors,
+      {
+        display: 'flex'
+      }
```

Comment:
Please pull this out into a static object so the benefits of memoization come into play. Right now this passes a new object each render.